pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/client/tabs/PentahoTab.java

```diff
@@ -148,10 +148,6 @@ public String getLabelTooltip() {
   }
 
   protected void closeTab() {
-    if ( getContent().getElement().getElementsByTagName( "iframe" ).getLength() > 0 ) {
```

Comment:
In that case, then move the fireCloseTab(..) method as well. If that method is only used from within the MantleTabPanel, then make it part of that class - it then has no reason to exist here.